README.md

```diff
@@ -1,4 +1,32 @@
-# sarif-pattern-matcher
+<p align="center"><img src="Src/img/header_img.png" width="440" /></p>
```

Comment:
i think we should keep this as markdown instead of merge markdown and HTML stuff.

Not sure if the header img is the best. Let's maintain the text for now

Reply:
I replaced the header just now if you think it work then ok otherwise I remove the header

Reply:
If you can remove for now. We will *probably* use the same icom from sarif-sdk.

Reply:
I remove the header part for now Thankyou for your ```review``` sir ☺️

Reply:
thank you.

Reply:
@virtualvivek, can you change the HTML part to Markdown style?

Reply:
> @virtualvivek, can you change the HTML part to Markdown style?

Hi @eddynaka sir, i have updated the markdown ``` html to pure markdown style```